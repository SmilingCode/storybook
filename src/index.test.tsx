import React from 'react'
import renderer from 'react-test-renderer'

const DemoBox = ({ text }: { text: string }) => {
  return <div>{text}</div>
}

test('DemoBox loads', () => {
  const component = renderer.create(<DemoBox text='woot' />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
