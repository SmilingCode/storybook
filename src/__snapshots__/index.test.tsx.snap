// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DemoBox loads 1`] = `
<div>
  woot
</div>
`;
